apps/emqx_resource/src/emqx_resource_manager.erl

```diff
@@ -406,6 +416,22 @@ handle_event(EventType, EventData, State, Data) ->
     ),
     keep_state_and_data.
 
+verify_state_consistency(State, #data{status = State} = Data) ->
```

Comment:
Nit:  mb? `log_state_consistency`

Reply:
Done in [d44e683](https://github.com/emqx/emqx/pull/10123/commits/d44e683088b9264ed0531f446c398d57924034bb).